example/lib/main.dart

```diff
@@ -5,11 +5,11 @@ import 'package:flutter/services.dart';
 import 'package:native_mixpanel/native_mixpanel.dart';
 
 void main() => runApp(MyApp(
-  mixpanel: Mixpanel(
-    shouldLogEvents: true,
-    isOptedOut: false,
-  ),
-));
+      mixpanel: Mixpanel(
```

Comment:
Please remove this indentation changes.

Reply:
Thank you for review. I will do it. It made autoformat of Flutter.